tests/setup.js

```diff
@@ -1,16 +1,8 @@
-import chai from 'chai';
-import chaiAsPromised from 'chai-as-promised';
 import { config } from 'dotenv';
 
 // Load environment variables from `.env`
 config();
```

Comment:
Is this needed now we have `setupFiles: ['dotenv/config', '<rootDir>/tests/setup.js']` ?

Reply:
Ah maybe needed for this file. Could move `setup.js` to `setupFilesAfterEnv` though perhaps?

Reply:
Good point. I'll remove the dotenv mention, but keep the file (as this is a template after all)

Reply:
1025b09e48a7ba5e6bf4d0221f9c9b12fc54003a